segmentationRDS/image.py

```diff
@@ -94,6 +157,7 @@ def writeImage(imagePath: str, image: np.ndarray, h_tgt: int, w_tgt: int, pixelA
     if imagePath[-4:].lower() == ".exr":
         output_image_spec.attribute('compression', 'zips') # required to get zip (1 scanline) compression method
     output_image_spec.attribute('pixelAspectRatio', pixelAspectRatio)
+    output_image_spec.attribute('Orientation', orientation)
```

Comment:
Is the standard Orientation flag with an upper case?

Reply:
Yes it is.